java-symbol-solver-testing/src/test/resources/javassist_symbols/README.md

```diff
@@ -0,0 +1,35 @@
+For the tests regarding resolving symbols in Jar files, we need some jar files. Some other tests within JavaSymbolSolver use established external jars for that purpose. Given the very specific cases we need here, that would severly complicate writing and maintaining tests. Therefore, I've decided to write custom jars for the cases we need.
+
+`main_jar` contains most of the necessary classes, `included_jar` and `excluded_jar` both contain an interface and a superclass. Included_jar is included in the combined type solver, while excluded_jar is not.
```

Comment:
Maybe in the future we can get Gradle tasks for that. For now this seems brilliant!

Reply:
Currently looking into this, as I needed to add some more files and was tired of executing the commands manually. Seems like it should be more or less possible.